src/backend/renderer/multigpu/egl.rs

```diff
@@ -67,7 +68,11 @@ impl<R: From<GlesRenderer> + Renderer<Error = GlesError>> GraphicsApi for EglGle
     type Device = EglGlesDevice<R>;
     type Error = Error;
 
-    fn enumerate(&self, list: &mut Vec<Self::Device>) -> Result<(), Self::Error> {
+    fn enumerate(
+        &self,
+        list: &mut Vec<Self::Device>,
+        capabilities: &Option<Vec<Capability>>,
```

Comment:
Could it be desirable to use different capabilities on different devices?

Reply:
Maybe, but this was about as complex as I was interested in making the multigpu changes.

I can also remove them for now and have multigpu support added by someone that makes use of that feature.